cypress/e2e/browse.js

```diff
@@ -32,9 +32,7 @@ describe('Browse page', function() {
 
         it('Should have download button', function() {
             cy.contains('Download')
-              .should('have.attr', 'href')
-              .and('include', 'super-studies')
-              .and('include', 'format=csv');
```

Comment:
I'm not sure why this test needs to change – can you enlighten me?

Reply:
@SandyRogers this test was previously checking that the download button contained certain values in it's href attribute, as shown in the code below. This test is no longer viable since we are no longer relying on the href, but rather using an onclick handler. 
I plan to write other tests to test the positive and negative cases of downloading private studies and analysis. 
However for now I thought it would better to first deploy the fix so the end user can access their downloads. 
This is the previous code I was referring to:

        it('Should have download button', function() {
            cy.contains('Download')
              .should('have.attr', 'href')
              .and('include', 'super-studies')
              .and('include', 'format=csv');
        });

Reply:
Gotcha - thanks for explaining! I misread the cypress call chain.